db/mysql/migrations/01_init/migration.sql

```diff
@@ -83,23 +83,17 @@ CREATE TABLE `event_data` (
     `event_id` VARCHAR(36) NOT NULL,
     `website_event_id` VARCHAR(36) NOT NULL,
     `website_id` VARCHAR(36) NOT NULL,
-    `session_id` VARCHAR(36) NOT NULL,
-    `url_path` VARCHAR(500) NOT NULL,
-    `event_name` VARCHAR(500) NOT NULL,
     `event_key` VARCHAR(500) NOT NULL,
-    `event_string_value` VARCHAR(500) NOT NULL,
-    `event_numeric_value` DECIMAL(19, 4) NOT NULL,
+    `event_string_value` VARCHAR(500) NULL,
+    `event_numeric_value` DECIMAL(19, 4) NULL,
     `event_date_value` TIMESTAMP(0) NULL,
     `event_data_type` INTEGER UNSIGNED NOT NULL,
```

Comment:
Postgres doesn't have unsigned ints. Better to just leave it as int for consistency. We're never gonna hit the limit.

Reply:
This is mysql.